tests/unit/PeopleAndRoles.spec.ts

```diff
@@ -108,7 +108,8 @@ describe('People And Roles component for Correction', () => {
     const router = mockRouter.mock()
     store.state.stateModel.tombstone.entityType = 'BEN'
     store.state.stateModel.tombstone.filingType = 'correction'
-    // store.state.stateModel.correctedFiling = { incorporationApplication: {} }
+    store.state.stateModel.entitySnapshot = {}
+    store.state.stateModel.correctedFiling = { incorporationApplication: {} }
```

Comment:
`correctedFiling` is now obsolete.

I think I will accept and merge this PR and then fix further in my PR.